auth/facebook.js

```diff
@@ -0,0 +1,49 @@
+const axios = require('axios')
+const router = require('express').Router()
+const { User } = require('../db/models')
+module.exports = router
+
+if (!process.env.FACEBOOK_APP_ID || !process.env.FACEBOOK_APP_SECRET) {
+
+  console.log('Facebook app ID / secret not found. Skipping Facebook OAuth.')
+
+} else {
+
+  router.get('/', (req, res, next) => {
+    console.log('got to facebook auth')
+    res.send('got to facebook auth')
+  })
+
+  router.post('/', (req, res, next) => {
+    console.log('incoming token:', req.body.token)
+    axios.get(`https://graph.facebook.com/me?fields=name,id,email,picture&access_token=${req.body.token}`)
```

Comment:
We need to handle the case when we pass an invalid access token. You should probably send back a 401 Unauthorized code.